javascript.ru/2.12.5/2.12.5.js

```diff
@@ -0,0 +1 @@
+﻿result(a + b < 4) ? 'few' : 'much';
```

Comment:
Can you rewrite example from the task with `alert(result);` in the end?
And then another way to do it (operator ?) with `alert(result);` in the end.
And read again the article. You didn't understand how it works and how to use.

Reply:
Done 87d4e07.